src/Network/Network/Cortex/ExpressRouteGateway/SetAzureRmExpressRouteGatewayCommand.cs

```diff
@@ -116,7 +118,7 @@ public override void Execute()
                 throw new PSArgumentException(Properties.Resources.ExpressRouteGatewayNotFound);
             }
 
-            if (this.MinScaleUnits > 0 && this.MaxScaleUnits > 0)
+            if (this.MinScaleUnits > 0 || this.MaxScaleUnits > 0)
```

Comment:
if one of these parameters was not present, can you please confirm whether line 128/129 still work?

Reply:
Yes, it will set the parameter to 0, which is the expected behavior.